eahub/templates/eahub/group.html

```diff
@@ -54,5 +54,44 @@ <h1>{{ group.name }}</h1>
       {% include 'eahub/maps/profile_page_map.html' with lat=group.lat|default_if_none:0 lon=group.lon|default_if_none:0 %}
     </div>
   </div>
+
+  <div class="claim_group_container">
+  {% if user.is_authenticated %}
+  {% if messages %}
+  <div class="col-md-6 col-md-offset-3">
+      {% for message in messages %}
+      <div {% if message.tags %} class="alert alert-{{message.tags}} hub-{{message.tags}} alert alert-dismissable"{% endif %}>
+        <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <div>{{ message }}</div>
+      </div>
+      {% endfor %}
+  </div>
+  {% else %}
+  <div class="text-center">
+    <div class="btn-group btn-group-sm text-center" role="group" id="claim_group_toggle">
+      <button type="button" class="btn btn-default claim_group_toggler"><i class="fa fa-sitemap"></i> Claim this group</button>
+    </div>
+    <div id="claim_group" class="col-md-6 col-md-offset-3 alert alert-info hub-info" role="alert">
+      <button type="button" class="close claim_group_toggler">
+        <span aria-hidden="true">&times;</span>
+      </button>
+      <p>If you are the organiser of this group, you can claim it so that you can edit it.
```

Comment:
Do we need this verbiage? Seems like we could just have either a list of organizers, or else just one sentence "We don't know who organizes this group." followed by a button "I organize this group".

Reply:
made it shorter